src/main/java/org/owasp/html/HtmlPolicyBuilder.java

```diff
@@ -288,7 +288,28 @@ public HtmlPolicyBuilder allowWithoutAttributes(String... elementNames) {
     }
     return this;
   }
-
+  	
+  /**
+   * Disallows the given element from appearing without the given attribute.
+   */
+  public HtmlPolicyBuilder disallowWithoutAttribute(String elementName, final String attributeName) {
```

Comment:
Right now, we can say

```java
    myPolicyBuilder.allowAttributes("src", ...).onElements("img")
```

and if I understand your goal, that is problematic because it allows but does not require `src="..."` on `<img ...>`.

I'd prefer

```java
    myPolicyBuilder.withAttributes("src", ...).required().onElements("img")
```

which allows mixing `required()` into the existing flow by which elements are associated with attributes instead of creating a new API.